jax/_src/numpy/lax_numpy.py

```diff
@@ -4067,6 +4067,25 @@ def polyint(p, m=1, k=None):
     return true_divide(concatenate((p, k)), coeff)
 
 
+@_wraps(np.polydiv)
+def polydiv(u,v):
+  _check_arraylike("polydiv", u, v)
+  u, v = _promote_dtypes_inexact(u, v)
+
+  w = u[0] + v[0]
+  m = len(u) - 1
+  n = len(v) - 1
+  scale = 1. / v[0]
+  q = zeros((_max(m - n + 1, 1),), w.dtype)
+  r = u.astype(w.dtype)
+  for k in range(0, m-n+1):
+    d = scale * r[k]   
+    q = q.at[k].set(d)
+    r = r.at[k:k+n+1].add(-d*v)
+  r = r.at[~(abs(r) > 1E-14).cumsum().astype(bool)].set(0)
```

Comment:
Maybe replace `1E-14` with `something * jnp.finfo(r.dtype).eps`?

Reply:
got it will update

Reply:
also, what will you suggest to make this code not outdated

Reply:
> what will you suggest to make this code not outdated

That I have no idea. 🤷